var/spack/repos/builtin/packages/umoci/package.py

```diff
@@ -23,20 +23,14 @@ class Umoci(MakefilePackage):
     version("0.4.1", sha256="0d83e01167383f529d726e9fd455660d4837371d5f0d82fad405f3ae6ae52486")
     version("0.4.0", sha256="66997e270dee8abc9796385b162a1e8e32dd2ee2359e5200af4e6671cc1e76a0")
```

Comment:
```suggestion
    version("0.4.0", sha256="66997e270dee8abc9796385b162a1e8e32dd2ee2359e5200af4e6671cc1e76a0")
```
Failed with go-1.23
```py
go: go.mod file not found in current directory or any parent directory; see 'go help modules'
```
This may be possible to solve using a version conflict on newer go compilers (but worth it?)

Please also see the check checksum errors that I got.